### PR TITLE
ThreatVault new endpoints - API

### PR DESCRIPTION
**Pull request by Ni-Knight**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-10961

## Description
Updating threat vault integration addin the new IP commands.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/PaloAltoNetworks_Threat_Vault/Integrations/ThreatVaultv2/ThreatVaultv2.yml [hunk 1]
```diff
@@ -56,11 +56,62 @@ configuration:
...
+      description: The actual score.
+      type: Number
+    - contextPath: DBotScore.Type
+      description: The indicator type.
+      type: String
+    - contextPath: DBotScore.Indicator
+      description: The indicator that was tested.
+      type: String
+    - contextPath: IP.Address
+      description: The IP Address.
```
Comment (ShirleyDenkberg) [inline Packs/PaloAltoNetworks_Threat_Vault/Integrations/ThreatVaultv2/ThreatVaultv2.yml:86]:
```suggestion
      description: The IP address.
```

File: Packs/PaloAltoNetworks_Threat_Vault/Integrations/ThreatVaultv2/ThreatVaultv2.yml [hunk 2]
```diff
@@ -56,11 +56,62 @@ configuration:
...
+      description: The IP address.
+      type: String
+    - contextPath: ThreatVault.IP.asn
+      description: The IP ASN.
+      type: String
+    - contextPath: ThreatVault.IP.geo
+      description: The country where the IP is located.
+      type: String
+    - contextPath: ThreatVault.IP.release.first_release_version
+      description: The release version when the IP Feed was released.
```
Comment (ShirleyDenkberg) [inline Packs/PaloAltoNetworks_Threat_Vault/Integrations/ThreatVaultv2/ThreatVaultv2.yml:104]:
```suggestion
      description: The release version when the IP feed was released.
```

File: Packs/PaloAltoNetworks_Threat_Vault/Integrations/ThreatVaultv2/ThreatVaultv2.yml [hunk 3]
```diff
@@ -56,11 +56,62 @@ configuration:
...
+      type: String
+    - contextPath: ThreatVault.IP.release.first_release_time
+      description: The release time when the IP Feed was released.
```
Comment (ShirleyDenkberg) [inline Packs/PaloAltoNetworks_Threat_Vault/Integrations/ThreatVaultv2/ThreatVaultv2.yml:107]:
```suggestion
      description: The release time when the IP feed was released.
```

File: Packs/PaloAltoNetworks_Threat_Vault/Integrations/ThreatVaultv2/ThreatVaultv2.yml [hunk 4]
```diff
@@ -56,11 +56,62 @@ configuration:
...
+      type: String
+    - contextPath: ThreatVault.IP.name
+      description: The name of the IP Feed entry.
+      type: String
+    - contextPath: ThreatVault.IP.status
+      description: The status of the IP Feed entry.
```
Comment (ShirleyDenkberg) [inline Packs/PaloAltoNetworks_Threat_Vault/Integrations/ThreatVaultv2/ThreatVaultv2.yml:113]:
```suggestion
      description: The name of the IP feed entry.
      type: String
    - contextPath: ThreatVault.IP.status
      description: The status of the IP feed entry.
```

File: Packs/PaloAltoNetworks_Threat_Vault/Integrations/ThreatVaultv2/ThreatVaultv2.yml [hunk 5]
```diff
@@ -1140,96 +1191,96 @@ script:
       description: The status of the threat (e.g., inactive, active, or released).
       type: String
   - arguments:
-    - description: 'Provides the ATP report by matching the report id '
+    - description: Provides the ATP report by matching the report id.
```
Comment (ShirleyDenkberg) [inline Packs/PaloAltoNetworks_Threat_Vault/Integrations/ThreatVaultv2/ThreatVaultv2.yml:1194]:
```suggestion
    - description: Provides the ATP report by matching the report ID.
```

File: Packs/PaloAltoNetworks_Threat_Vault/Integrations/ThreatVaultv2/ThreatVaultv2.yml [hunk 6]
```diff
@@ -1140,96 +1191,96 @@ script:
...
       isArray: true
       name: report_id
       required: true
-    description: Retrieve the Advanced Threat Prevention (ATP) report by report id in batch mode. Batch limit is 100 entries. Get one or more ATP reports. Must provide one or more report IDs
+    description: Retrieve the Advanced Threat Prevention (ATP) report by report id in batch mode. Batch limit is 100 entries. Get one or more ATP reports. Must provide one or more report IDs.
```
Comment (ShirleyDenkberg) [inline Packs/PaloAltoNetworks_Threat_Vault/Integrations/ThreatVaultv2/ThreatVaultv2.yml:1198]:
```suggestion
    description: Retrieve the Advanced Threat Prevention (ATP) report by report ID in batch mode. Batch limit is 100 entries. Get one or more ATP reports. Must provide one or more report IDs.
```

File: Packs/PaloAltoNetworks_Threat_Vault/Integrations/ThreatVaultv2/ThreatVaultv2.yml [hunk 7]
```diff
@@ -1239,10 +1290,10 @@ script:
     name: threatvault-atp-report-pcap-get
     outputs:
     - contextPath: ThreatVault.ATP.PCAP.ID
-      description: Threatvault ATP PCAP ID
+      description: Threatvault ATP PCAP ID.
       type: string
     - contextPath: ThreatVault.ATP.PCAP.Name
-      description: Threatvault ATP PCAP Name
+      description: Threatvault ATP PCAP Name.
```
Comment (ShirleyDenkberg) [inline Packs/PaloAltoNetworks_Threat_Vault/Integrations/ThreatVaultv2/ThreatVaultv2.yml:1296]:
```suggestion
      description: Threatvault ATP PCAP name.
```

File: Packs/PaloAltoNetworks_Threat_Vault/ReleaseNotes/2_0_14.md [hunk 8]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+
+##### Palo Alto Networks Threat Vault v2
+Updated the enricher to support the new ip-feed api end-point.
```
Comment (ShirleyDenkberg) [inline Packs/PaloAltoNetworks_Threat_Vault/ReleaseNotes/2_0_14.md:5]:
```suggestion
Updated the enricher to support the new ip-feed API end-point. 
```

**Comment by ShirleyDenkberg** [comment]
@ShahafBenYakir @MLainer1 Doc review completed.

**Review by samuelFain — APPROVED** [review]
LGTM, Approved.
Please address my comments before merging.

**Pull request merged**